pkg/config/resources.go

```diff
@@ -37,6 +37,10 @@ const (
 	ExternalAttacherClusterRoleBinding    ResourceName = "external-attacher-clusterrolebinding"
 	ExternalSnapshotterClusterRole        ResourceName = "external-snapshotter-clusterrole"
 	ExternalSnapshotterClusterRoleBinding ResourceName = "external-snapshotter-clusterrolebinding"
+	SCCForControllerClusterRole           ResourceName = "scc-controller"
+	SCCForControllerClusterRoleBinding    ResourceName = "scc-controller"
+	SCCForNodeClusterRole                 ResourceName = "scc-node"
+	SCCForNodeClusterRoleBinding          ResourceName = "scc-node"
```

Comment:
You are not creating an SCC here, but assign clusterRole and ClusterRoleBidnings, correct?  The Resource names should be:
<serviceAccount>-<node/controller>-<clusterrole/clusterrolebinding>

Maybe the funtion name should be AddSCCTo<role> as that would map more closely to the oc adm command.
oc adm policy add-scc-to-user